examples/v1/taskruns/alpha/stepaction-results.yaml

```diff
@@ -6,8 +6,10 @@ spec:
   image: alpine
   results:
     - name: result1
+    - name: result2
   script: |
     echo "I am a Step Action!!!" >> $(step.results.result1.path)
+    echo "I am a hidden step action!!!" >> $(step.results.result2.path)
```

Comment:
Can you explain why do we need a second result here?

Reply:
Just to show that the task only extracts one of the two.